lib/Fetcher/FeedFetcher.php

```diff
@@ -294,11 +298,15 @@ protected function buildItem(
         $item->setGuidHash(md5($item->getGuid()));
 
         $lastModified = $parsedItem->getLastModified() ?? new DateTime();
-        if ($parsedItem->getValue('pubDate') !== null) {
-            $pubDT = new DateTime($parsedItem->getValue('pubDate'));
-        } elseif ($parsedItem->getValue('published') !== null) {
-            $pubDT = new DateTime($parsedItem->getValue('published'));
-        } else {
+        try {
+            if ($parsedItem->getValue('pubDate') !== null) {
+                $pubDT = new DateTime($parsedItem->getValue('pubDate'));
+            } elseif ($parsedItem->getValue('published') !== null) {
+                $pubDT = new DateTime($parsedItem->getValue('published'));
+            } else {
+                $pubDT = $lastModified;
+            }
```

Comment:
Isn't this a duplicate with the catch?

Reply:
Aye, `$pubDT = $lastModified;` could be moved after line 300, then we do not need the else branch and can have the catch block empty.